kiosk/kiosk_browser/main_widget.py

```diff
@@ -13,8 +15,17 @@ class MainWidget(QtWidgets.QWidget):
     def __init__(self, kiosk_url: str, settings_url: str, toggle_settings_key: str):
         super(MainWidget, self).__init__()
 
+        if platform.system() in ['Darwin']:
+            from kiosk_browser.proxy import Proxy
+            import os
+            system = System(name = "PlayOS",
+                            version = os.getenv("PLAYOS_VERSION","1.0.0-dev"))
+        else:
+            from kiosk_browser.dbus_proxy import DBusProxy as Proxy
+            system = System()
```

Comment:
What if we moved this branching to the `system` module/`System` class?

It could offer something like a `def infer() -> System` function.

Reply:
Something like this?

[Move platform logic to system module](https://github.com/dividat/playos/pull/167/commits/4213269a2466a873fab377093013d222df494e44)